### DISCUSSION TITLE
Version rollback on Home Assistant Install

### DISCUSSION THREAD

**Discussion by CobraDunn** [mKeRix/room-assistant#753] [Q&A]
I love room-assistant but it has become unstable in recent updates.  How do I roll back to previous versions of room-assistant on my Home Assistant OS installation?

**Comment by warez1975** [comment]
> I love room-assistant but it has become unstable in recent updates. How do I roll back to previous versions of room-assistant on my Home Assistant OS installation?

try:

`sudo npm i --global --unsafe-perm room-assistant@VERSION`

here you can find the releases:

https://github.com/mKeRix/room-assistant/releases
